qdrant-landing/content/documentation/guides/distributed_deployment.md

```diff
@@ -516,6 +486,97 @@ Physical shards require a large amount of resources, so make sure your custom sh
 
 For large cardinality keys, it is recommended to use [partition by payload](/documentation/guides/multiple-partitions/#partition-by-payload) instead.
 
+Now you need to create custom shards:
```

Comment:
```suggestion
Now you need to create custom shards ([API reference](https://api.qdrant.tech/api-reference/distributed/update-collection-cluster#request.body.CreateShardingKeyOperation.create_sharding_key)):
```

Reply:
Linked https://api.qdrant.tech/api-reference/distributed/create-shard-key#request since we used dedicated API:

```
PUT /collections/{collection_name}/shards
{
  "shard_key": "{shard_key}"
}
```